cmake/app.cmake

```diff
@@ -1,15 +1,9 @@
 # SPDX-License-Identifier: Apache-2.0
 
-# Set cached global first
-# TODO: @mbed-os-tools - toolchain should come from tools
-set(MBED_TOOLCHAIN "GCC_ARM" CACHE STRING "")
-# TODO: @mbed-os-tools - target should come from tools
-set(MBED_TARGET "K64F" CACHE STRING "")
-
-include(${MBED_ROOT}/cmake/toolchain.cmake)
-include(${MBED_ROOT}/cmake/target.cmake)
-include(${MBED_ROOT}/cmake/env.cmake)
-include(${MBED_ROOT}/cmake/util.cmake)
+include(gen_config/CMakeLists.txt)
+include(${MBED_OS_ROOT}/cmake/toolchain.cmake)
```

Comment:
`MBED_ROOT`

Reply:
Godd catch

btw, I understand with this is necessary in `include` clauses, why do we need `${MBED_ROOT}` in the other cmake clauses ? (paths are relative to `.` any way)